test/testutils.js

```diff
@@ -709,28 +709,29 @@ var commons;
     }
   }
 
-  testUtils.resultsDeepEqual = (
-    obj1,
-    obj2,
-    ignoredPaths = [],
+  testUtils.assertResultsDeepEqual = (
+    expected,
+    actual,
+    ignoredPaths = [
+      'timestamp',
+      // testEnvironment is ignored by default because Chrome's UI animations for the
+      // "an automated test is controlling this browser" notification can cause
+      // inconsistencies in windowHeight for otherwise-identical scans.
```

Comment:
Thank you for adding this comment. I didn't understand why we were ignoring it before 👍